Gemfile

```diff
@@ -0,0 +1,46 @@
+source 'https://rubygems.org'
```

Comment:
Over time, I would remove the commented out code in here, however it's fine to leave if you are actually referencing it.

Reply:
will remove the commented lines